space_commands/space_launches.json

```diff
@@ -0,0 +1 @@
13.733","longitude":"80.235","location":{"id":14,"url":"https://ll.thespacedevs.com/2.2.0/location/14/","name":"Sriharikota, Republic of India","country_code":"IND","map_image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launch_images/location_14_20200803142403.jpg","total_launch_count":53,"total_landing_count":0},"map_image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launch_images/pad_50_20200803143457.jpg","total_launch_count":38},"webcast_live":false,"image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launcher_images/sslv_image_20190830213312.jpg","infographic":null,"program":[]},{"id":"54c50fe1-bbd1-4515-9630-37612383c3c1","url":"https://ll.thespacedevs.com/2.2.0/launch/54c50fe1-bbd1-4515-9630-37612383c3c1/","launch_library_id":1983,"slug":"soyuz-21afregat-m-cas500-1-rideshare","name":"Soyuz 2.1a/Fregat-M | CAS500-1 & rideshare","status":{"id":2,"name":"TBD"},"net":"2021-03-20T00:00:00Z","window_end":"2021-03-20T00:00:00Z","window_start":"2021-03-20T00:00:00Z","inhold":false,"tbdtime":true,"tbddate":false,"probability":-1,"holdreason":"","failreason":"","hashtag":null,"launch_service_provider":{"id":63,"url":"https://ll.thespacedevs.com/2.2.0/agencies/63/","name":"Russian Federal Space Agency (ROSCOSMOS)","type":"Government"},"rocket":{"id":2541,"configuration":{"id":45,"launch_library_id":83,"url":"https://ll.thespacedevs.com/2.2.0/config/launcher/45/","name":"Soyuz 2.1a/Fregat-M","family":"Soyuz","full_name":"Soyuz 2.1a Fregat-M","variant":"Fregat-M"}},"mission":{"id":1246,"launch_library_id":null,"name":"CAS500-1 & rideshare","description":"CAS500-1 is the first of two South Korean Earth observation satellites. These spacecraft feature the AEISS-C imaging system with a ground resolution of 0.5 m in panchromatic mode and 2 m in color mode.\r\n\r\nMore than a dozen other satellites are included as secondary payloads, among them being the first spacecraft by the Catalan Space Agency.","launch_designator":null,"type":"Earth Science","orbit":{"id":17,"name":"Sun-Synchronous Orbit","abbrev":"SSO"}},"pad":{"id":20,"url":"https://ll.thespacedevs.com/2.2.0/pad/20/","agency_id":null,"name":"31/6","info_url":null,"wiki_url":"","map_url":"http://maps.google.com/maps?q=45.996+N,+63.564+E","latitude":"45.996034","longitude":"63.564003","location":{"id":15,"url":"https://ll.thespacedevs.com/2.2.0/location/15/","name":"Baikonur Cosmodrome, Republic of Kazakhstan","country_code":"KAZ","map_image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launch_images/location_15_20200803142517.jpg","total_launch_count":466,"total_landing_count":0},"map_image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launch_images/pad_20_20200803143516.jpg","total_launch_count":76},"webcast_live":false,"image":"https://spacelaunchnow-prod-east.nyc3.digitaloceanspaces.com/media/launcher_images/soyuz25202.1a_image_20190726062341.png","infographic":null,"program":[]}]}
```

Comment:
Maybe this file should be in the .gitignore?

Reply:
I'll just put the whole space_commands in .gitignore.